lte/gateway/python/magma/enodebd/devices/freedomfi_one.py

```diff
@@ -233,7 +233,8 @@ def set_magma_device_cfg(
         success_str = "SUCCESS"  # String constant returned by radio
         insync_str = "INSYNC"
 
-        if name_to_val[cls.DEFAULT_GW].upper() != success_str:
+        if name_to_val.get(cls.DEFAULT_GW) \
+                and name_to_val[cls.DEFAULT_GW].upper() != success_str:
```

Comment:
```suggestion
        if (name_to_val.get(cls.DEFAULT_GW) and 
             name_to_val[cls.DEFAULT_GW].upper()) != success_str:
```
wrap this (and the other usages) in parentheses to get rid of the linting warnings

Reply:
As discussed I will fix this on master.